README.md

```diff
@@ -2,6 +2,16 @@
 
 ## How to request support for a new language track
 
+**Does the language qualify for having a track?**
+
+If you can answer yes to the following:
+
+- It is a Turing-complete programming language, so no regular expressions or web framework tracks.
```

Comment:
Now I'm waiting for a TeX track…

Reply:
Yeah, in writing this criterion, various ways to abuse it also came to my mind. I think as long as we're open honest proposals, e.g. a TeX track that focuses on TeX programming isn't necessarily a bad idea. Another example is that I have an idea for a [`jq`](https://github.com/stedolan/jq) track: `jq` is mainly a query language, but it is Turing-complete, and you can write functions and importable modules.

In these fringe cases, I think it is better to resolve whether they should be included based on conversation and evaluation rather than find perfect criteria; these criteria are only for "should I even apply?" for those who are unsure if their idea is a fit at all, and is meant for easing the path towards track request applications.

Reply:
Late to the party, but yeah: I'd prefer to resolve edge cases with discussion. E.g. I am not going to allow a CSS track even though I'm told it's Turing complete.